doc/source/getting_started/user_guide.rst

```diff
@@ -1,33 +1,30 @@
-===============
-Getting started
-===============
+==========
+User guide
+==========
 
-To run the Antenna Toolkit, you must have a licensed copy of AEDT installed.
 You have multiple options for installing and launching this toolkit:
 
-- You can install the toolkit directly in AEDT via an installation script and then launch it
-  as a wizard. For more information, see :ref:`install-toolkit-AEDT`.
-- You can install the toolkit from the AEDT console and then launch it as a wizard. For more
-  information, see :ref:`install_toolkit_console_ui`.
-- You can install and launch the toolkit directly from a Python console and then use the toolkit's APIs.
+- You can install the toolkit directly in AEDT via an installation script and then launch it as a wizard.
+  For more information, see :ref:`install-toolkit-AEDT`.
+- You can install the toolkit from a Python console and then launch Antenna wizard.
+  For more information, see :ref:`install_toolkit_console_ui`.
+- You can install the toolkit from a Python console and then use the toolkit's APIs.
   For more information, see :ref:`install_toolkit_console_api`.
 
 .. _install-toolkit-AEDT:
 
-How to install directly in AEDT and launch as a wizard
-------------------------------------------------------
+Install toolkit in AEDT and launch the Antenna wizard
+-----------------------------------------------------
 
 You can install the Antenna Toolkit directly in AEDT using the base
 interpreter from the AEDT installation.
 
-#. From `Install from a Python file <https://aedt.docs.pyansys.com/version/stable//Getting_started/Installation.html#install-from-a-python-file>`_
-   in the PyAEDT documentation, download the ``PyAEDTInstallerFromDesktop.py`` file and then run this Python script.
+#. From `Install from a Python file <https://aedt.docs.pyansys.com/version/stable/Getting_started/Installation.html#install-from-a-python-file>`_,
```

Comment:
You should always tell a user where the link is going to take them so that they can decide if they want to click it. This is why the "in the PyAEDT documentation was there. I'd suggest keeping it there.